src/components/Grid/Grid.test.tsx

```diff
@@ -54,24 +71,26 @@ describe("Grid", () => {
   };
 
   const renderGrid = ({
-    rowCount = 20,
-    columnCount = 20,
+    rowCount,
+    columnCount,
     columnWidth,
     onColumnResize,
     focus,
     onFocusChange,
+    rowAutoHeight,
     ...props
   }: Props) =>
     renderCUI(
       <Grid
-        rowCount={rowCount}
-        columnCount={columnCount}
+        rowCount={rowCount ?? 20}
+        columnCount={columnCount ?? 20}
```

Comment:
Why this change? Setting the values as default arguments will apply that to the whole function, rather than in just these places. Do we not want that here?

Reply:
Sorry, was testing something and forgot to change back!